wire-library/wire-kotlin-generator/src/test/java/com/squareup/wire/kotlin/KotlinGeneratorTest.kt

```diff
@@ -147,25 +147,29 @@ class KotlinGeneratorTest {
   }
 
   @Test fun blockingSingleRequestSingleResponse() {
+    //language=kotlin
     val expected = """
-          |package routeguide
-          |
-          |import com.squareup.wire.GrpcClient
-          |import com.squareup.wire.Service
-          |
-          |/**
-          | * RouteGuide service interface.
-          | */
-          |class RouteGuideBlockingServer(
-          |  private val client: GrpcClient
-          |) : Service {
-          |  /**
-          |   * Returns the [Feature] for a [Point].
-          |   */
-          |  fun GetFeature(request: Point): Feature {
-          |  }
-          |}
-          |""".trimMargin()
+        package routeguide
+        
+        import com.squareup.wire.Service
+        import com.squareup.wire.WireRpc
+        
+        /**
+         * RouteGuide service interface.
+         */
+        interface RouteGuideBlockingServer : Service {
+          /**
+           * Returns the [Feature] for a [Point].
+           */
+          @WireRpc(
+            path = "/routeguide.RouteGuide/GetFeature",
+            requestAdapter = "routeguide.Point#ADAPTER",
+            responseAdapter = "routeguide.Feature#ADAPTER"
+          )
+          fun GetFeature(request: Point): Feature
+        }
+        
+        """.trimIndent()
```

Comment:
Was the code written without margin before too? Reads nicely. Wonder if that can confuse anyone that this is actually a string and not real code when reading in the middle of it.

Reply:
This has changed from `trimMargin()` with pipe prefixes to `trimIndent()`. I don't think it's confusing.